parts/linux/cloud-init/artifacts/cse_helpers_mariner.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+echo "Sourcing cse_helpers_distro.sh for Mariner"
```

Comment:
all these methods are used by Ubuntu as well. Why do you need to move it out for Mariner?